test/integration/coscheduling_test.go

```diff
@@ -248,9 +376,19 @@ func TestCoschedulingPlugin(t *testing.T) {
 			testCtx.Scheduler.SchedulingQueue.MoveAllToActiveOrBackoffQueue("fake event")
 
 			err = wait.Poll(1*time.Second, 120*time.Second, func() (bool, error) {
+				notFound := true
 				for _, v := range tt.expectedPods {
-					if !podScheduled(cs, ns, v) {
+					if !podScheduled(cs, ns.Name, v) {
 						t.Logf("waiting pod failed %v", v)
+						if !tt.allowPodsNotIn {
```

Comment:
Please re-evaluate if `allowPodsNotIn` is needed. If it is, add a comment explaining the intention.